src/util/strconv.h

```diff
@@ -56,4 +57,13 @@ static int ParseBoolean(const char *arg, int *res) {
   return 0;
 }
 
+static char *strtolower(char *str) {
```

Comment:
So its static and you are using it from outside? I do not think you are using the right one ..

Reply:
Also its not ok that you are changing the char* return from the RedisModuleString, there is a reason why it const.

Reply:
Still static? not follow how come you use it from outside?

Reply:
It is a header file which I include directly